galasa-ui/src/components/test-runs/results/TestRunsTable.tsx

```diff
@@ -144,22 +138,33 @@ export default function TestRunsTable({
    * This component encapsulates the logic for rendering a cell.
    * It renders a special layout for the 'result' column and a default for all others.
    */
-  const CustomCell = ({ header, value }: CustomCellProps) => {
-    let cellComponent = <TableCell>{value}</TableCell>;
+  const CustomCell = ({ header, value, href }: CustomCellProps) => {
+    let cellComponent = (
+      <TableCell className={styles.linkCell}>
+        {value}
+        <a href={href} className={styles.linkOverlay} />
+      </TableCell>
+    );
 
     if (value === 'N/A' || !value) {
       return <TableCell>N/A</TableCell>;
     }
 
     if (header === 'result') {
       cellComponent = (
-        <TableCell>
+        <TableCell className={styles.linkCell}>
           <StatusIndicator status={value as string} />
+          <a href={href} className={styles.linkOverlay} />
         </TableCell>
       );
     } else if (header === 'submittedAt') {
       // Format the date using the context's formatDate function
-      cellComponent = <TableCell>{formatDate(new Date(value))}</TableCell>;
+      cellComponent = (
+        <TableCell className={styles.linkCell}>
+          {formatDate(new Date(value))}
+          <a href={href} className={styles.linkOverlay} />
+        </TableCell>
```

Comment:
The `<a>` tags seem to be making the screen flash when clicking on a row, which is more visible when using dark mode.

You could try using Next.js' `<Link>` tag with `prefetch={false}` to avoid pre-fetching all of the test run detail pages. See https://nextjs.org/docs/app/api-reference/components/link